### ISSUE TITLE
ECS Step - Unable to save process with non-default Health Check Interval - Container health check's interval value of container "mycontainer" is invalid.

### ISSUE THREAD

**Issue by FinnianDempsey** [OctopusDeploy/Issues#7447]
### Team

- [X] I've assigned a team label to this issue

### Severity

Workaround Exists

### Version

Effects Latest Version

### Latest Version

I could reproduce the problem in the latest build

### What happened?

When defining a Container Health check in the ECS Step, if a non-default value is used for the _Container -> Heath Check -> Interval_ then the process is unable to be saved. An error appears indicating that the value for the Interval cannot contain the name used by the container.

### Reproduction

1. Configure ECS Step with a container Health Check
2. Adjust the Health Check Interval
3. See Error 

### Error and Stacktrace

```shell
Container health check's interval value of container "tentacle" is invalid.
```

### More Information

<img width="593" alt="Screen Shot 2022-03-24 at 10 37 42" src="https://user-images.githubusercontent.com/78527975/159819067-3ba2289a-d7c3-455f-9630-4914220d1c44.png">
<img width="1614" alt="Screen Shot 2022-03-24 at 10 37 25" src="https://user-images.githubusercontent.com/78527975/159819075-99fde8d4-3730-4f9c-855b-2a796dc59f90.png">
<img width="558" alt="Screen Shot 2022-03-24 at 10 39 09" src="https://user-images.githubusercontent.com/78527975/159819242-52b1d31c-316e-4e96-ada3-93c0f5840506.png">

### Workaround

Use the default value of 0 or extract the CloudFormation Template to use in another step.

**Comment by scottmerchant** [comment]
Hi Finnian, in your screenshot you specify a value of 1 but the info text below the input field says between 5 - 300. Can you try a value in that range and let us know?

**Comment by FinnianDempsey** [comment]
Ahh that was it! 

Interval values of 5+ don't have this error

Could this error message be improved, or remove the ability to input incorrect values for this parameter?

**Comment by scottmerchant** [comment]
I've created a PR to improve the error messages for this value (and other health check values) as it's a relatively easy win.

**Comment by geofflamrock** [comment]
This was fixed in v1.3.0 of the `Deploy Amazon ECS Step` which is available to all customers who have step template updates enabled. That version is also available from Octopus 2022.3.1854+ if step template updates are not enabled.
